### ISSUE TITLE
Automatic renewal of ssl certificates

### ISSUE THREAD

**Issue by ltetrel** [neurolibre/neurolibre-binderhub#12]
We should clarify who is responsible for this ??
Some answers here : https://community.letsencrypt.org/t/how-to-automatically-renew-certificates/4393/4

**Comment by lunebellec** [comment]
agreed, we need to ensure this is done in a timely manner. To be discussed on the next neurolibre group call.

**Comment by ltetrel** [comment]
https://www.beyondtrust.com/docs/remote-support/how-to/sslcertificates/create-ca-signed.htm

**Comment by ltetrel** [comment]
Actually its pretty simple with cloudflare, it's just 5$ a month... So we need to talk about it next meeting don't know who is on charge of that....
![image](https://user-images.githubusercontent.com/37963074/67521653-4c1dae00-f679-11e9-8d9e-bcdfb1e31308.png)
